docs/azure/azure-how-to/instances/launch-ubuntu-images.rst

```diff
@@ -1,161 +1,132 @@
     az group create \
-        -n $RESOURCE_GROUP_NAME \
-        -l $LOCATION
+        --name "$RESOURCE_GROUP_NAME" \
+        --location "$LOCATION"
 
 
-Create a Virtual Machine
-~~~~~~~~~~~~~~~~~~~~~~~~
+Create a virtual machine
+------------------------
 
-Create a virtual machine using the `vm create`_ command.
-
-Depending on the type of Ubuntu image selected, additional arguments may be required to correctly launch
-the image. However, most Ubuntu images can be launched with the default arguments:
+Create a virtual machine using the `vm create`_ command. Virtual machines on Azure are created with `Trusted Launch`_
+enabled by default. All Hyper-V Gen2 x86 Ubuntu images from 20.04 LTS forward support Trusted Launch and can be created
+using the command defaults:
 
 .. code::
 
     az vm create \
-        --name $VIRTUAL_MACHINE_NAME \
-        --resource-group $RESOURCE_GROUP_NAME \
-        --image $UBUNTU_IMAGE_URN \
+        --resource-group "$RESOURCE_GROUP_NAME" \
+        --location "$LOCATION" \
+        --name "$VIRTUAL_MACHINE_NAME" \
+        --image "$UBUNTU_IMAGE_URN" \
         --generate-ssh-keys
 
-If you want to launch images with security features enabled, refer to the following sections.
-
-
-With Trusted Launch
-+++++++++++++++++++
-
-All Ubuntu images from Ubuntu 20.04 LTS (Focal Fossa) forward support Trusted Launch on Hyper-V Gen2 SKUs. Example
-definitions of Ubuntu image URNs for Hyper-V Gen2 include:
+ARM64 images
+~~~~~~~~~~~~
 
-.. code::
-
-    UBUNTU_IMAGE_URN="Canonical:ubuntu-24_04-lts:server:latest"
-    # or
-    UBUNTU_IMAGE_URN="Canonical:0001-com-ubuntu-server-jammy:22_04-lts-gen2:latest"
-
-Define the variables to set the security type:
+ARM64 images must be launched with an ARM64-compatible virtual machine size. Furthermore, ARM64 images do not support
+Trusted Launch and must instead use the **Standard** security type. You may also need to register usage of the security
+type for your Azure subscription:
 
 .. code::
 
-    SECURITY_TYPE=TrustedLaunch
+    az feature register \
+        --namespace "Microsoft.Compute" \
+        --name "UseStandardSecurityType"
 
-Create the virtual machine:
+    # Required to propagate the changes.
+    az provider register --name "Microsoft.Compute"
 
-.. code::
+    # VM size should contain the "p" additive feature denoting an ARM64 processor.
+    VIRTUAL_MACHINE_SIZE="Standard_D2ps_v5"
+    SECURITY_TYPE="Standard"
 
     az vm create \
-        --name $VIRTUAL_MACHINE_NAME \
-        --resource-group $RESOURCE_GROUP_NAME \
-        --image $UBUNTU_IMAGE_URN \
-        --security-type $SECURITY_TYPE \
+        --resource-group "$RESOURCE_GROUP_NAME" \
+        --location "$LOCATION" \
+        --name "$VIRTUAL_MACHINE_NAME" \
+        --image "$UBUNTU_IMAGE_URN" \
+        --security-type "$SECURITY_TYPE" \
+        --size "$VIRTUAL_MACHINE_SIZE" \
         --generate-ssh-keys
 
 
-With Confidential VM
-++++++++++++++++++++
+Confidential Virtual Machines (CVM)
```

Comment:
In the interest of cohesion, I believe we should either rename the `ARM images` header to `ARM machines` or rename this header to `Confidential Virtual Machine (CVM) images`. Either both concern images or both concern machines.